docs/development/getting_started.rst

```diff
@@ -90,17 +96,25 @@ Install the dependencies for the development environment:
 #. VirtualBox_
 #. Ansible_.
 
-There are several ways to install Ansible on a Mac. We recommend setting up a virtual environment to install Ansible since we are currently using Ansible 1.8.4:
+There are several ways to install Ansible on a Mac. We recommend installing it
+to a virtual environment using ``virtualenvwrapper`` and `pip`, so as not to
```

Comment:
sorry just caught this when doing a test build of the docs: double backtick for fixed width font needed on ``pip``:

<img width="295" alt="screen shot 2017-02-01 at 5 48 51 pm" src="https://cloud.githubusercontent.com/assets/7832803/22534106/b8c6b8d2-e8a6-11e6-9320-b83fb4780b9e.png">